src/memcache/gc.cpp

```diff
@@ -122,10 +122,25 @@ void gc_thread::gc() {
         return false;
     };
 
+    // Putting the thread to sleep tens of microseconds with each loop is desired, but due to the precision of the timer,
+    // it's not appropriate to do sleep with each loop. The `initial_repl_sleep_delay` is accumulated until it
+    // exceeds 10000 microseconds (10 milliseconds), and then the thread is put to sleep all at once when this limit is exceeded.
+    static const std::uint64_t SLEEP_THRESHOLD = 10000;
```

Comment:
I guess `static` does not make much sense here.

Reply:
IIUC, it is an idiom to make it compile-time constant. Should we use constexpr instead for readability?

Reply:
hmm... too difficult for me.
https://stackoverflow.com/questions/177437/what-does-const-static-mean-in-c-and-c

Reply:
anyway, I guess that does not make a big difference.
please feel free to ignore my comment.

Reply:
Thank you. Fixed in 36f6e2d.